src/openthaigpt_pretraining/utils/data_cleaning.py

```diff
@@ -0,0 +1,10 @@
+
+
+def clean_text(line: str) -> str:
+    # () is remainder after link in it filtered out
+    return line.strip().replace("\n", " ").replace("()", "")
+
+def filter_out_line(line: str) -> str:
+    if len(line) < 80:
```

Comment:
Will remove all lines that are less than 80, will the model perform bad on short sentence?

Reply:
I remove it but we can custom preprocess in the future